### ISSUE TITLE
Toggle size adjust not working, can't change fov.

### ISSUE THREAD

**Issue by Jellyfishsandwich** [whateverusername0/VRTRAKILL#178]
I cant access the mod submenu its missing. I've tried to change the fov in the config files but it only changes the view on the desktop not my headset. everything is too big and fov too narrow. in the config it says press "J" for toggle size adjust but it does nothing. I have tried both x64 and x86 bepinex to no avail also tried disabling steam world scale adjust and still nothing. anyone know how to fix this so I can change fov and toggle size adjust/world scale? any help appreciated.
Win 10 x64
Quest 2

**Comment by whateverusername0** [comment]
yeah it's just there... hoping someone will fix that since i lowkey stopped working on the mod

**Comment by AmeliaGL0w0** [comment]
> I cant access the mod submenu its missing. I've tried to change the fov in the config files but it only changes the view on the desktop not my headset. everything is too big and fov too narrow. in the config it says press "J" for toggle size adjust but it does nothing. I have tried both x64 and x86 bepinex to no avail also tried disabling steam world scale adjust and still nothing. anyone know how to fix this so I can change fov and toggle size adjust/world scale? any help appreciated. Win 10 x64 Quest 2

you cant change the fov in your headset, as its a hardware limit. unless you meant to change the hud size, so it doesnt end up in the far edge of your perephial vision, then im not sure

**Comment by Jellyfishsandwich** [comment]
I managed to find a workaround using steam vr beta settings for worldscale (0.7 looks about right to me), however I still cant access any of the vr mods options.
I can see the menu additions when I look at the code and they just aren't ingame when I compile and run it. I get fpv and motion controls in vr just the mods menu items are missing now. thanks for the reply before whateverusername0 this mod is sick bro I hope sometime you pick it up again... do you have a patroen? I will donate something at the end of the month.
